pbandb/ft_p
# Feature(client): read multiline welcome message

Now the initial read over PI socket can handle multiple lines from server.

Test it:
- Start pure-ftp container as usual: `docker-compose -f ./resources/pureftpd/docker-compose.yml up -d`
- Run `./client 0.0.0.0 21`; welcome message and prompt should be displayed
